### PR TITLE
dev

### PR DESCRIPTION
**Pull request by Ryphier**

### CODE REVIEW

**Review by realashleybailey — CHANGES_REQUESTED** [review]
The placeholder attribute specifies a short hint that describes the expected value of an input field (e.g. a sample value). We have labels to specify input type, please undo these changes and resubmit.

File: app/templates/wizard/requests.html
Comment (realashleybailey) [inline app/templates/wizard/requests.html:1]: This does not need changing.

File: app/templates/wizard/jellyfin/tips.html
Comment (realashleybailey) [inline app/templates/wizard/jellyfin/tips.html:1]: You removed punctuation for the title, "Eh, So, What is Jellyfin exactly?" is fine.

We can not name Netflix specifically, please keep it generalised. 
Paragraph 1 can be "Jellyfin is an opensource platform that lets you stream movies and TV shows in one place.
        It's like having your own personal movie theater right at your fingertips! Think of it as a digital library of
        your favorite content that you can access from anywhere, on any device - your phone, tablet, laptop, smart TV,
        you name it.""

Paragraph 2 is fine
Link is fine too

File: app/templates/wizard/jellyfin/download.html
Comment (realashleybailey) [inline app/templates/wizard/jellyfin/download.html:1]: Remove "(it works great!) or another device"

File: app/templates/wizard/jellyfin/tips.html
Comment (realashleybailey) [reply app/templates/wizard/jellyfin/tips.html:1]: Why on earth do we need a bloody orangutan 😂

**Comment by Ryphier** [comment]
Hahaha shit! I Didn't mean to pull this!

**Pull request closed without merging**